media-gfx/nomacs/nomacs-3.17.2282.ebuild

```diff
@@ -0,0 +1,91 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PLUGIN_PKG="${PN}-plugins-$(ver_cut 1-2)"
+inherit cmake xdg-utils
+
+DESCRIPTION="Qt-based image viewer"
+HOMEPAGE="https://nomacs.org/"
+SRC_URI="https://github.com/${PN}/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz
+plugins? ( https://github.com/${PN}/${PN}-plugins/archive/$(ver_cut 1-2).tar.gz -> ${PLUGIN_PKG}.tar.gz )"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~riscv ~x86 ~amd64-linux"
+IUSE="+jpeg +opencv plugins raw +tiff zip"
+
+REQUIRED_USE="
+	raw? ( opencv )
+	tiff? ( opencv )
+"
+
+RDEPEND="
+	dev-qt/qtconcurrent:5
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5[jpeg?]
+	dev-qt/qtnetwork:5
+	dev-qt/qtprintsupport:5
+	dev-qt/qtsvg:5
+	dev-qt/qtwidgets:5
+	media-gfx/exiv2:=
+	opencv? ( >=media-libs/opencv-3.4:= )
+	raw? ( media-libs/libraw:= )
+	tiff? (
+		dev-qt/qtimageformats:5
+		media-libs/tiff:=
+	)
+	zip? ( dev-libs/quazip:0=[qt5(+)] )
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	dev-qt/linguist-tools:5
+	virtual/pkgconfig
+"
+
+S="${WORKDIR}/${P}/ImageLounge"
+
+DOCS=( src/changelog.txt )
+
+PATCHES=(
+	"${FILESDIR}"/${P}-exiv-0.28.0-build-fix.patch
+)
+
+src_prepare() {
+	if use plugins ; then
+		rmdir plugins || die
+		mv -v ../../${PLUGIN_PKG} plugins || die
+		# Fix bug #847112
+		sed -e 's:QT_QMAKE_EXECUTABLE NAMES "qmake" "qmake-qt5" "qmake.exe":QT_QMAKE_EXECUTABLE NAMES "qmake" "qmake5" "qmake.exe":' \
+			-i plugins/cmake/Utils.cmake || die
+		# Fix nomacs-plugins installation and search library directory
+		sed -e "s:lib/nomacs-plugins:$(get_libdir)/nomacs-plugins:" \
+			-i plugins/cmake/Utils.cmake || die
+		sed -e "s:lib/nomacs-plugins:$(get_libdir)/nomacs-plugins:" \
+			-i src/DkCore/DkPluginManager.cpp || die
```

Comment:
This could be replaced with upstream commit: https://github.com/nomacs/nomacs/commit/c8f6c325c5f5170bb473c41ee716b781d37bdd41

Of course not without adjusting paths and dropping unrelated patch hunks (version bump, subproject)